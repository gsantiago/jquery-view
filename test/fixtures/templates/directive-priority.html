<ul>
  <li :repeat="item in items"
      :class="{active: (item === 'Active Item')}"
      :bind="item">
  </li>
</ul>