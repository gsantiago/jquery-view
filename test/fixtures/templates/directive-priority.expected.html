<ul>
  <li>First Item</li><li class="active">Active Item</li><li>Last Item</li>
</ul>